README.md

```diff
@@ -1,31 +1,179 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-# Linters Test
-> This project is to test linters
-## Built With
-- HTML and CSS
+<a name="readme-top"></a>
 
-## Getting Started
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖To Do list: list structure](#to-do-list-list-structure)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖To Do list: list structure
+
+ <a name="about-project"></a>
+
+> This is A simple but effective and responsive (mobile first) Microverse exercize project.
+>
+> **o-do list** is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> I am using Html, Css and Javascript es6 Front end development.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JAVASCRIPT</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Some key feature of this project
+
+- **User can add Book by adding book title and book author]**
+- **User can Remove Book from book list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> ## Our project demo picture
+> <img align="center" src="project_demo.PNG" alt="logo" width="100%"  height="auto" />
+  <br/>
```

Comment:
- [ ] Kindly fix your logo here or remove it.

  ![dgdhf](https://user-images.githubusercontent.com/36542986/207308014-8f20b488-da3f-4851-b516-783da19b1004.jpg)